data-model/fdo-types/0.3.0/digital-specimens/schema/identification.json

```diff
@@ -0,0 +1,375 @@
+    },
+    "dwc:identificationID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationID",
+      "examples": [
+        "9992"
+      ]
+    },
+    "ods:identificationType": {
+      "type": "string",
+      "description": "The type of identification"
+    },
+    "ods:taxonFormula": {
+      "type": "string",
+      "description": "The full formula of the taxonomic identification"
+    },
+    "dwc:verbatimIdentification": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/verbatimIdentification",
+      "examples": [
+        "Peromyscus sp."
+      ]
+    },
+    "dwc:typeStatus": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/typeStatus",
+      "examples": [
+        "holotype"
+      ]
+    },
+    "dwc:identifiedBy": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identifiedBy",
+      "examples": [
+        "James L. Patton"
+      ]
+    },
+    "dwc:identifiedByID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identifiedByID",
+      "examples": [
+        "https://orcid.org/0000-0002-1825-0097"
+      ]
+    },
+    "dwc:dateIdentified": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/dateIdentified",
+      "examples": [
+        "2009-02-20T08:40Z"
+      ]
+    },
+    "dwc:identificationReferences": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationReferences",
+      "examples": [
+        "Aves del Noroeste Patagonico. Christie et al. 2004."
+      ]
+    },
+    "ods:isVerifiedIdentification": {
+      "type": "boolean",
+      "description": "If this is the accepted identification, based on https://rs.tdwg.org/dwc/terms/identificationVerificationStatus",
+      "examples": [
+        true
+      ]
+    },
+    "dwc:identificationRemarks": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationRemarks",
+      "examples": [
+        "Distinguished between Anthus correndera and Anthus hellmayri based on the comparative lengths of the uñas."
+      ]
+    },
+    "dwc:identificationQualifier": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/identificationQualifier",
+      "examples": [
+        "cf. var. oxyadenia"
+      ]
+    },
+    "ods:typeDesignationType": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "ods:typeDesignatedBy": {
+      "type": "string",
+      "description": "Unclear yet",
+      "$comment": "Unknown what this field should be"
+    },
+    "ods:hasCitation": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "https://schemas.dissco.tech/schemas/digitalobjects/0.1.0/shared-models/citation.json"
+      }
+    },
+    "ods:hasTaxonIdentification": {
```

Comment:
how can one identification have multiple taxonIdentification. is this meant to indicate multiple species in a specimen? I would expect it would go like:

specimen -> one or more Identifications per specimen -> one taxon identification per Identification

how do multiple taxonIdentifiations relate to the field ods:isVerifiedIdentification? What does it mean if a varified Identification has multiple taxonIdentifications, and they conflict?

Reply:
Ok good questions. So let's say we have a fur had and the fur had contains parts of multiple species. We could get one identification (which is accepted) containing multiple taxonIdentifications (one for each species). That is the one example I would use. But now let's say I don't agree with you I add another identification with maybe different list of taxonidentifications. So then we have 2 Identification each with let's say 4 taxonIdentifications. 
Sure we need to figure this out in the code and interface but we would have room for it in the model.

Reply:
Ok i see what you're saying. I think it makes more sense to have isVerifiedIdentification on the taxonIdentification and not the group of taxonIdentifications but if acceptedIdentification is on the class then that's just the granularity we work with.